docker/Dockerfile.php

```diff
@@ -0,0 +1,70 @@
+FROM php:7.1.33-fpm AS base
+WORKDIR /var/www/html
+
+RUN apt update -y
+RUN DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
```

Comment:
Since these are intended for production I think here an in any other added Dockerfile we should be very explicit which versions of dependencies we use so containers do not break without any changes here down the line.

I pushed a commit to your branch which performs that cleanup, and also adds linting for the added Dockerfiles.